test/Sentry.Tests/Protocol/SdkVersionTests.cs

```diff
@@ -57,5 +58,41 @@ public static IEnumerable<object[]> TestCases()
             sdk.AddPackage("a", "1");
             yield return new object[] { (sdk, "{\"packages\":[{\"name\":\"a\",\"version\":\"1\"},{\"name\":\"b\",\"version\":\"2\"}]}") };
         }
+
+        [Fact]
+        public void SerializeObject_IgnoresDuplicatePackages()
+        {
+            var sdkVersion = new SdkVersion
+            {
+                Name = "Sentry.Test.SDK",
+                Version = "3.9.2"
+            };
+            sdkVersion.AddPackage("Foo", "Alpha");
+            sdkVersion.AddPackage("Bar", "Beta");
```

Comment:
Not sure if I follow, but shouldn't you insert a duplicated package into sdkVersion?
```suggestion
            sdkVersion.AddPackage("Bar", "Beta");
            sdkVersion.AddPackage("Bar", "Beta");
```

Reply:
🤦🏻‍♂️ Yes... I somehow removed that when cleaning this up. Good catch lol